content/handbook/security/corporate/end-user-services/laptop-management/_index.md

```
@@ -0,0 +1,89 @@
+
+より高性能なマシンを必要とするほとんどのロールは、14 インチまたは 16 インチの MacBook Pro パフォーマンスモデルが承認されています。どのマシンの対象となるかを判断するには、この[スプレッドシート](https://docs.google.com/spreadsheets/d/1409j22VpvOwF7DrVGhvJ4PJmwboG0NjoMu6UKx8YXfE/edit?usp=sharing)（公開）を参照してください。
+
+### Linux ハードウェア
+
+{{% alert color="warning" %}}
+Linux は macOS の代替として利用可能ですが、サポートが限定的なため推奨はされていません。Linux を選択する場合、自分の環境を維持する完全な責任を引き受け、IT がサポートを提供しないことを理解する必要があります。Slack に Tips やコツを交換するための #linux チャンネルがありますが、公式のヘルプデスクリソースではありません。
+{{% /alert %}}
+
+**Dell が唯一承認された Linux ノートパソコンベンダーです**。これらのノートパソコンは一般的に Ubuntu Linux がプリインストールされて出荷されます。一部の国では Dell は Linux プリインストールのノートパソコンを販売していないため、その場合チームメンバーが Linux を自分でインストールする必要があります。
+
+> Linux ノートパソコンの最大価格は、**同等の [16 インチ MacBook Pro ノートパソコン](#apple-hardware) の価格**を超えてはなりません。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)